.storybook/storiesDisplay.scss

```diff
@@ -0,0 +1,3 @@
+.wrapper {
+    min-height: 225px;
+}
```

Comment:
curious mixture of some files with trailing newline, and others like this without. Usually not important, but i have a teeny tiny preference for trailing newlines to prevent UNIX weirdness.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Oh yeah the formatting is quite broken on all of this

I'm very tempted to move the `commitizen` & `eslint` config packages to GitHub so we can use them here 🤔 

but that's not super necessary

Reply:
Will definitely pay attention to the newlines getting added